styles/typography.css

```diff
@@ -16,3 +15,90 @@
   --font-family-serif: "Source Serif 4", serif;
   --bs-font-sans-serif: "Source Sans 3";
 }
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  font-weight: 600;
+}
+
+h6,
+.h6 {
+  text-transform: uppercase;
+}
+
+dt {
+  font-weight: 600;
+}
+
+dd {
+  font-weight: 400;
+}
+
+p {
+  font-size: 1rem;
```

Comment:
Why is these needed? Aren't these the defaults?

Reply:
🤷🏻 they were specified in the ticket. I can take it out if we want to just rely on browser default